_data/api.yml

```diff
@@ -1758,6 +1758,9 @@ sections:
           pool:
             type: Option[String]
             description: If specified, only addresses associated with the given pool are deleted. Otherwise, all addresses are deleted.
+          address:
+            type: Option[String]
+            description: If specified, delete only the given address.
```

Comment:
perhaps something like:
If specified, delete only the given address, else all of them.

and modify:
description: Delete all addresses associated with a given tag, optionally associated with a pool
to be something like:
description: Delete addresses associated with a given tag, optionally associated with a pool, or optional single address